arex-schedule-web-api/src/main/java/com/arextest/schedule/progress/impl/UpdateResultProgressEventImpl.java

```diff
@@ -320,4 +321,8 @@ public void onActionCaseLoaded(ReplayActionItem actionItem) {
     LOGGER.info("update the replay action case count, action id:{} , size: {}", actionItem.getId(),
         actionItem.getReplayCaseCount());
   }
+
+  @Override
+  public void onReplayCaseFinish(ReplayActionCaseItem actionItem) {
+  }
```

Comment:
It is recommended to define an empty implementation of the method directly using the default keyword